dbt_common/events/logger.py

```diff
@@ -13,6 +13,8 @@
 from dbt_common.events.format import timestamp_to_datetime_string
 from dbt_common.utils.encoding import ForgivingJSONEncoder
 
+from dbt_common.events.types_pb2 import PrintEvent
```

Comment:
It would be nice if we could avoid importing from types_pb2 if we can, because those classes don't always behave like "real" Python classes and it would be nice to isolate the importing of those classes to just the matching xxx_types.py files. Could we look at msg.info.name instead?  It should be the string "PrintEvent".

Reply:
Thanks!! This also makes it easier if we need to do some expansion.